File: test_xcode_Framework/test_project/test_project/ViewController.m
//
//  ViewController.m
//  test_project
//
//  Created by Aka on 2017/8/12.
//  Copyright © 2017年 Aka. All rights reserved.
//

#import "ViewController.h"
#import "Test1ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
	Test1ViewController *vc = [Test1ViewController new];
    [self.navigationController pushViewController:vc animated:YES];

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: test_snapkit/test_snapkit/ViewController.swift
//
//  ViewController.swift
//  test_snapkit
//
//  Created by Aka on 2018/5/30.
//  Copyright © 2018年 Aka. All rights reserved.
//

import UIKit

import SnapKit

class ViewController: UIViewController {
    lazy var box = UIView()

    override func viewDidLoad() {
        super.viewDidLoad()
        self.view.addSubview(box)
        box.snp.makeConstraints { (make) in
            make.width.height.equalTo(50)
            make.center.equalTo(self.view)
        }
        box.backgroundColor = UIColor.red
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}

